playbooks/roles/githubsshkeys/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+- name: install httplib2
+  sudo: true
+  pip: name=httplib2
+
+- name: get SSH keys from GitHub
+  uri: >
+    url=https://github.com/{{ item }}.keys
+    return_content=true
+  register: ssh_keys
+  with_items: add_ssh_keys_from_github.usernames
+
+- name: add SSH key labels to ~/.ssh/authorized_keys
+  lineinfile: >
+    dest=~/.ssh/authorized_keys
```

Comment:
`~` will refer to a different directory depending on the user running the playbook. We should define a variable called `githubsshkeys_user`, then [become](https://docs.ansible.com/ansible/become.html) that user for this task and the next one.